src/biokbase/narrative/jobs/jobcomm.py

```diff
@@ -99,42 +99,43 @@ class JobRequest:
     The IDs of the job(s) to perform the function on.
 
     """
+
     def __init__(self, rq: dict):
         self.raw_request = copy.deepcopy(rq)
         self.msg_id = rq.get("msg_id")  # might be useful later?
         self.rq_data = rq.get("content", {}).get("data")
         if self.rq_data is None:
-            raise ValueError(INVALID_REQUEST_ERR)
+            raise JobRequestException(INVALID_REQUEST_ERR)
```

Comment:
Change anything related to incorrect job request structure/format into a `JobRequestException`